### ISSUE TITLE
The bower main section Should use non minified version for correct wiredep behaviour

### ISSUE THREAD

**Issue by mhawila** [m-e-conroy/angular-dialog-service#135]
When using Grunt wiredep task to wire dependencies I notice the version I pulled from bower (version 5.2.8)  had the main section specify the minified version (see below) contrary to "standard" practice. I had to override the property in my bower.json file for it to work properly. 

"main": [
    "dist/dialogs.min.css",
    "dist/dialogs.min.js",
    "dist/dialogs-default-translations.min.js"
  ],

Wouldn't it be nice to modity it to follow the conventions and become

"main": [
    "dist/dialogs.css",
    "dist/dialogs.js",
    "dist/dialogs-default-translations.js"
  ],

**Comment by dougmoscrop** [comment]
I agree - for now you can override this in your own `bower.json`. I think we should make this a major version change since theoretically some peoples build pipeline might now be assuming that it is minified?
